src/Forms/Shared/Samples/GraphicsOverlay/DictionaryRendererGraphicsOverlay/DictionaryRendererGraphicsOverlay.xaml.cs

```diff
@@ -52,14 +53,21 @@ private async void Initialize()
                 // Prevent graphics from showing up when zoomed too far out.
                 _tacticalMessageOverlay.MinScale = 1000000;
 
-                // Create a symbol dictionary style following the mil2525d spec.
-                string symbolFilePath = DataManager.GetDataFolder("c78b149a1d52414682c86a5feeb13d30", "mil2525d.stylx");
-                DictionarySymbolStyle mil2525DStyle = await DictionarySymbolStyle.CreateFromFileAsync(symbolFilePath);
+                // Create the dictionary symbol style from the Joint Military Symbology MIL-STD-2525D portal item.
+                var symbolStyleUri = new Uri("https://www.arcgis.com/home/item.html?id=d815f3bdf6e6452bb8fd153b654c94ca");
+                DictionarySymbolStyle dictionarySymbolStyle = await DictionarySymbolStyle.OpenAsync(symbolStyleUri);
 
-                // Use the dictionary symbol style to render graphics in the overlay.
-                _tacticalMessageOverlay.Renderer = new DictionaryRenderer(mil2525DStyle);
+                // Find the first configuration setting which has the property name "model", and set its value to "ORDERED ANCHOR POINT".
+                if (dictionarySymbolStyle?.Configurations?.Where(config => config.Name == "model").FirstOrDefault() is DictionarySymbolStyleConfiguration configuration)
+                {
+                    configuration.Value = "ORDERED ANCHOR POINT";
+                }
 
-                // Load the military messages and render them.
+                // Create a new dictionary renderer from the dictionary symbol style to render graphics with symbol dictionary attributes and set it to the graphics overlay renderer.
+                var dictionaryRenderer = new DictionaryRenderer(dictionarySymbolStyle);
+                _tacticalMessageOverlay.Renderer = dictionaryRenderer;
```

Comment:
I think this could be a single line, I don't see the variable used anywhere else.
```suggestion
                _tacticalMessageOverlay.Renderer = new DictionaryRenderer(dictionarySymbolStyle);
```